stochastic_physics_mpas.F

```diff
@@ -255,19 +279,36 @@ subroutine stochastic_physics_pattern_apply(domain, num_tends, tend_names, ierr)
       print*,'There are more than one block, in this MPI rank',  domain%dminfo%my_proc_id  
     endif
 
- ! get AR(1) pertubation_pattern
-
- ! retrieve the specified tendencies that needs to be perturbed
-   call mpas_pool_get_subpool(block%structs,'tend_physics' ,tend_physics)
+ ! retrieve the specified tendencies that needs to be perturbed and
+ ! apply the patterns (at all levels) to the tendency      
    do i = 1, num_tends
      tend = tend_names(i)
-     select case (tend)
-       case ("xxx")
-         call mpas_pool_get_array(tend_physics,'xxx',tend_array)
- ! apply the patterns (at all levels) to the tendency      
+!     print*, "tendency from physics parameterization to be perturbed:", trim(tend)
+     select case (trim(tend))
+       case ("rucuten")
+         call mpas_pool_get_array(tend_physics,'rucuten',tend_array)
+         call apply_pattern(tend_array, stoch_pat)
+         if (pid == 0) print*, 'apply perturbation pattern to rucuten'
+       case ("rvcuten")
+         call mpas_pool_get_array(tend_physics,'rvcuten',tend_array)
+         call apply_pattern(tend_array, stoch_pat)
+         if (pid == 0) print*, 'apply perturbation pattern to rvcuten'
+       case ("rublten")
+         call mpas_pool_get_array(tend_physics,'rublten',tend_array)
+         call apply_pattern(tend_array, stoch_pat)
+         if (pid == 0) print*, 'apply perturbation pattern to rublten'
+       case ("rvblten")
+         call mpas_pool_get_array(tend_physics,'rvblten',tend_array)
+         call apply_pattern(tend_array, stoch_pat)
+         if (pid == 0) print*, 'apply perturbation pattern to rvblten'
+       case ("tend_rtheta_physics")
+         call mpas_pool_get_array(tend_physics,'tend_rtheta_physics',tend_array)
+         call apply_pattern(tend_array, stoch_pat)
+         if (pid == 0) print*, 'apply perturbation pattern to rtheta'
+       case ("tend_rho_physics")
+         call mpas_pool_get_array(tend_physics,'tend_rho_physics',tend_array)
```

Comment:
Should we leave the "rho" case here if it's never going to be perturbed?

Reply:
Following Ning's comments, I'm confirming we'll keep this in case it may be necessary in the future.